### ISSUE TITLE
Add the ability to customize the output window

### ISSUE THREAD

**Issue by dypa** [AnWeber/vscode-httpyac#15]
User story
As user would like to be able to configure output window.

In setting i see option Customize Response with options:
- see only response body
- see only response headers
- see response headers and body
- see request and response headers and body

**Comment by AnWeber** [comment]
The requirement basically comes from the implementation of [vscode-restclient](https://marketplace.visualstudio.com/items?itemName=humao.rest-client), which supports this setting. I implemented this relatively deliberately differently, since I preferred to use the functionality of vscode.
I use a text document directly for the body output. It is possible to display the headers as CodeLens or as Markdown. And additionally I output the response in the outputchannel httpyac in RFC2616 format. In the future I consider to support the notebook editor of vscode.
It is possible to add this display format with a custom extension. I already provide the necessary interfaces.If you are interested, I can support you and give you help for the first steps.

**Comment by AnWeber** [comment]
Here is the explanation of what steps would need to be implemented
1. import [extension api](https://code.visualstudio.com/api/references/vscode-api#extensions) and register custom [outputProcessor](https://github.com/AnWeber/httpyac/blob/main/src/output/httpOutputProcessor.ts)
```js
const httpyac= extensions.getExtension('vscode-httpyac');
let importedApi = httpyac.exports;
importedApi.httpYacApi.httpOutputProcessors.splice(0,1, customOutputProcessor);
```

2. create customOutputProcessor analog to [vscode-restclient](https://github.com/Huachao/vscode-restclient/blob/master/src/views/httpResponseWebview.ts)

**Comment by AnWeber** [comment]
My mistake. I notice that I also need this function more often than I expected. I added a new setting `httpyac.responseViewContent` with values body, headers, full, exchange.
A disadvantage is the missing auto-formatting of the response body by vscode.

**Comment by AnWeber** [comment]
released version 2.4.0 with this enhancement
